tests/integration/test_session.py

```diff
@@ -468,3 +468,25 @@ def test_session_auto_ip(mini_sentry, relay_with_processing, sessions_consumer):
     # Can't test ip_address since it's not posted to Kafka. Just test that it is accepted.
     session = sessions_consumer.get_session()
     assert session
+
+
+def test_session_invalid_release(mini_sentry, relay_with_processing, sessions_consumer):
+    relay = relay_with_processing()
+    sessions_consumer = sessions_consumer()
+
+    PROJECT_ID = 42
+    project_config = mini_sentry.add_full_project_config(PROJECT_ID)
+    project_config["config"]["eventRetention"] = 17
+
+    timestamp = datetime.now(tz=timezone.utc)
+    relay.send_session(
+        PROJECT_ID,
+        {
+            "sid": "8333339f-5675-4f89-a9a0-1c935255ab58",
+            "timestamp": timestamp.isoformat(),
+            "started": timestamp.isoformat(),
+            "attrs": {"release": "latest"},
```

Comment:
And maybe some tests for erroneous releases like the ones mentioned here -> https://github.com/getsentry/sentry/pull/26279/files#diff-21815168d041846d793a0e86869d5eb8391601025fb32017363e3359e93188b5R649

Reply:
Could do, but since this is in integration tests, I see little point in testing `sentry-release-parser`.